tests/drivers/counter/counter_basic_api/boards/nrf9160_pca10090.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_COUNTER_TIMER0=y
+CONFIG_COUNTER_TIMER1=y
+CONFIG_COUNTER_TIMER2=y
+CONFIG_COUNTER_RTC0=y
```

Comment:
Does this need a new line here,  @nordic-krch ?

Reply:
done